### ISSUE TITLE
Trickle issue with JAWS w.r.t. new incomplete content aria

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2513]
### Subject of the issue
JAWS reads “incomplete content above” with a full width assessment question, after completing an assessment question and moving to the next question.

### Environment
* Framework 4.3.0
* Trickle master
* JAWS2019 + IE11 + Windows 10

### Expected behaviour
The text “incomplete content above” should only read if trickle is locked.

Similar to #2135

**Comment by oliverfoster** [comment]
Cannot replicate using current [framework master](https://github.com/adaptlearning/adapt_framework/tree/25dae7acdc027a3747cd0f59bc75eb89976b5187) and current [trickle master ](https://github.com/adaptlearning/adapt-contrib-trickle/tree/2ffae8fe90849954bc80e9431e214059f6d63ecc)


Here's a recording from the screen reader: [sr-recording.zip](https://github.com/adaptlearning/adapt_framework/files/3590762/sr-recording.zip)

Tested on ie11+window10+jaws2019, using vanilla adapt framework without the assessment graphics and the mcqs full width as described.

![image](https://user-images.githubusercontent.com/7974663/64535878-f9c63f00-d30f-11e9-9061-dccd5c3896b4.png)

![image](https://user-images.githubusercontent.com/7974663/64535894-00ed4d00-d310-11e9-80e4-aecf505600f2.png)

**Comment by oliverfoster** [comment]
this should be fixed now
